src/views/Swap/Swap.vue

```diff
@@ -563,7 +564,13 @@ export default {
       return dpUI(min)
     },
     max () {
-      return this.available && !isNaN(this.available) ? BN.min(BN(this.available)) : BN(0)
+      const max = this.available && !isNaN(this.available) ? BN.min(BN(this.available)) : BN(0)
+
+      if (max && this.asset === 'UST') {
```

Comment:
This shouldn't be required if `LiqualitySwapProvider.estimateFees` is implemented correctly